src/main/scala/io/archivesunleashed/ArchiveRecordImpl.scala

```diff
@@ -29,6 +29,11 @@ import org.archive.io.arc.ARCRecord
 import org.archive.io.warc.WARCRecord
 import org.archive.util.ArchiveUtils
 
+/** ArchiveRecord.
+ *
+ *  @constructor an archive record.

```

Comment:
I believe the style guide calls for a noun phrase without trailing period?
http://www.oracle.com/technetwork/java/javase/tech/index-137868.html

Reply:
I had 
```### Description
+
+The `ArchiveRecord` [class](https://docs.scala-lang.org/tour/classes.html) is used by `RecordLoader` to manage the ingestion of WARC and ARC files. It represents one record from a collection found in the file. It uses the Internet Archive's ARCRecord & WARCRecord APIs to extract data from web crawls such as the crawl date, content strings, urls and so on.

`ArchiveRecord` is a generic record that covers data from both ARCRecord and WARCRecord format. AUT will convert either format into the generic `ArchiveRecord` class.``` in the original reference document.